deploy/lib/xquery/setup.xqy

```diff
@@ -3136,6 +3140,59 @@ declare function setup:validate-group(
       setup:validation-fail(fn:concat("Missing Group: ", $group))
 };
 
+declare function setup:validate-group-settings(
+  $group-config as element(gr:group),
+  $setting as element()
+) {
+  let $group := $group-config/gr:group-name/fn:string(.)
+  let $setting-name := $setting/fn:string(.)
+  let $function-name := fn:concat("group-get-", $setting-name, "s")
+  let $keys := fn:tokenize($setting/@keys, " ")
+  let $existing :=
+    try
+    {
+      xdmp:eval(
```

Comment:
I suppose but the same technique: ```try{xdmp:eval}catch(){}``` is used in many places in the same module. That's the reason I choose to reuse it.